packages/formatron/src/dataTypes.ts/enum.ts

```diff
@@ -0,0 +1,26 @@
+import {DataType} from '../dataType';
+
+export class EnumType extends DataType {
+  public static readonly typeName = 'enum';
+
+  isOfType(value: unknown): value is string {
+    return typeof value === 'string' ? this.field.name === value : false;
+  }
+
+  get isMulti(): boolean {
+    return this.field.options?.multi ?? false;
+  }
+
+  // TODO Figure out best way to implement:
```

Comment:
I think the best way to implement this is via that `DataInterface` concept.  The other way is to make Safety Suite's method of handling enums the "Formatron" way of doing things, but its not a great system imo.  Having it in the data interface allows different systems to being in enums from.. wherever they want based on an arbitrary `enumId` value.